bats/tests/snapshots/restore-snapshot-after-factory-reset.bats

```diff
@@ -45,8 +45,7 @@ local_setup() {
     # don't provide a --container-engine.name argument to `rdctl start`
     RD_CONTAINER_ENGINE=""
     # don't create settings.json file
-    RD_USE_PROFILE=true
-    start_kubernetes
+    RD_USE_PROFILE=true start_kubernetes
```

Comment:
I intentionally had this split into 2 lines, even though the variable is indeed only needed for this single command.

I find it somewhat more readable to have the actual command at the beginning of the line rather then hidden somewhere in the middle of it. I know sometimes it is hard to avoid, e.g. if you are calling external commands, and the variable would then need to be explicitly exported, but in this case just splitting the line and having the command on a line by itself is easy enough.

Reply:
I made it into one line to be clear that the variable doesn't leak into future lines. Otherwise we'd need to dig into the other commands to see if they're affected (they're not).

Reply:
It is a global setting, so if it was used by any other function, then it would have to be set globally anyways, so that is not really a concern.

We use the same pattern in other tests too, so they will now become inconsistent, unless you change them both.

And if I look at

```bash
@test 'start up with NO profiles' {
    RD_USE_PROFILE=false
    RD_USE_IMAGE_ALLOW_LIST=false
    start_application
}
```

I find this definitely clearer to read than

```bash
@test 'start up with NO profiles' {
    RD_USE_PROFILE=false RD_USE_IMAGE_ALLOW_LIST=false start_application
}
```